deployment/build-s3-dist.sh

```diff
@@ -43,14 +46,14 @@ echo "--------------------------------------------------------------------------
 echo "[Init] Remove any old dist files from previous runs"
 echo "------------------------------------------------------------------------------"
 
-echo "rm -rf $template_dist_dir"
-rm -rf $template_dist_dir
-echo "mkdir -p $template_dist_dir"
-mkdir -p $template_dist_dir
-echo "rm -rf $build_dist_dir"
-rm -rf $build_dist_dir
-echo "mkdir -p $build_dist_dir"
-mkdir -p $build_dist_dir
+echo "rm -rf $global_dist_dir"
+rm -rf $global_dist_dir
```

Comment:
I am paranoid and always surround variables in quotes, though I think this will work like you expect even if that path has spaces in it

Reply:
I'm def not an expert in shell scripting. this whole build script probably needs to be re-evaluated, but since updating the build process is a different story in the backlog I focused just on making sure the folder structure changes didn't break the existing build process.